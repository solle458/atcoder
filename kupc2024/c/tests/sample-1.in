3
scs
..c
.sc
