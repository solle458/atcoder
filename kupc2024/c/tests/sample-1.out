Yes
0.23670068 0.00000000
1.81649658 3.00000000
